meta-rauc-qemuarm/recipes-kernel/linux/linux-yocto_%.bbappend

```
@@ -0,0 +1,24 @@
+# ---
+# qemuarm specific kernel extensions and configurations
+#
+# - a custom device tree into the kernel to allow easy adjustment of the
+#   emulated peripherals and "hardware" properties
+# - provide kernel configuration required by RAUC
+
+FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
+
+SRC_URI += "\
+    file://qemuarm.dts \
+    file://rauc.cfg \
+"
+
+# Architecture specific kernel configuration
+KERNEL_IMAGETYPE:qemuarm = "zImage"
+KERNEL_DEVICETREE:qemuarm = "qemuarm.dtb"
+KERNEL_EXTRA_ARGS:qemuarm += "LOADADDR=${UBOOT_ENTRYPOINT}"
+
+# Instead of patching the DTS into the kernel tree, copy it to allow easier modification of the file.
+do_configure:append() {
+    cp ${WORKDIR}/qemuarm.dts ${S}/arch/arm/boot/dts
```

Just stumbled across this in a test build:

```suggestion
    cp ${UNPACKDIR}/qemuarm.dts ${S}/arch/arm/boot/dts
```
for master.

Just as an idea if moving to master is too much overhead: We could also apply this to `scarthgap` first and do a forward-port later (despite I also prefer backports of course).